extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,90 @@
+import Client from 'ad-engine/src/utils/client';
+import Context from 'ad-engine/src/services/context-service';
+import ScrollListener from 'ad-engine/src/listeners/scroll-listener';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import StringBuilder from 'ad-engine/src/utils/string-builder';
+import config from './context';
+
+Context.extend(config);
+ScrollListener.init();
+let supportedTemplates = [ BigFancyAdBelow ];
+
+supportedTemplates.forEach((template) => {
+	TemplateService.register(template);
+});
+
+function init(slotRegistry, pageLevelTargeting, skin) {
+	overrideSlotService(slotRegistry);
+	updatePageLevelTargeting(pageLevelTargeting, skin);
+}
+
+function overrideSlotService(slotRegistry) {
+	SlotService.getBySlotName = (id) => {
+		if (id) {
+			let slot = slotRegistry.get(id);
+			return unifySlotInterface(slot);
+		}
+	};
+}
+
+function unifySlotInterface(slot) {
+	const slotContext = Context.get(`slots.${slot.name}`) || {targeting: {}};
+	slot.getSlotName = () => slot.name;
+	slot.default = {
+		getSlotName: () => slot.name
+	};
+	slot.getId = () => slot.name;
+	slot.config = slotContext;
+	slot.getVideoAdUnit = () => buildVastAdUnit(slot.name);
+	slot.getTargeting = () => slotContext.targeting;
+	return slot;
+}
+
+function buildVastAdUnit(slotName) {
+	return StringBuilder.build(
+		Context.get(`vast.adUnitId`),
+		Object.assign({}, Context.get('targeting'), Context.get(`slots.${slotName}`))
+	);
+}
+
+function loadCustomAd(fallback) {
+	return (params) => {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Convert this ES6 arrow function into a regular one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AArrowFunctions)

[project-id]: wikia:app